pkg/helper/client_test.go

```diff
@@ -0,0 +1,99 @@
+package helper
```

Comment:
please add a test that we add another interface in the filter config.

check that we can filter both veth and vnet.